### PR TITLE
feat(my-profile): change avatar ready for test

### PR DESCRIPTION
**Pull request by DiLiUA**
- added crop-image component;
  - developed functionality for crop image;

### CODE REVIEW

File: src/my-profile/my-profile.component.ts [hunk 1]
```diff
@@ -11,21 +11,29 @@ import { UserService } from '../user-service/user.service';
 import { ChangableData } from './changable-data.model';
 import { customToastOptions } from '../shared/models/toasty-options.model';
 import { UpdateUserProfileRequestObject } from '../user-service/update-user-profile.interface';
+import { CropImageComponent } from '../shared/crop-image/crop-image.component';
+import { UploadFilesService } from '../shared/upload-files/upload-files.service';
+import { UploadFile } from '../shared/upload-files/upload-file.model';
```
Comment (Ugmaxie) [inline src/my-profile/my-profile.component.ts:16]: use common module + index to concatenate all you shared models, components and servises in order to simplify code =>
import { Model1, Model2 } from './shared/models';
import { Service1, Service2 } from './shared/service';

File: src/my-profile/my-profile.component.ts [hunk 2]
```diff
@@ -11,21 +11,29 @@ import { UserService } from '../user-service/user.service';
...
   styleUrls: ['./my-profile.component.scss']
 })
 export class MyProfileComponent implements OnDestroy {
+  @ViewChild('cropImgModal', undefined) cropper: CropImageComponent;
+
   subscriptionManager: Subscription = new Subscription();
   pattern = Pattern;
   currentUser: User;
   changableData: ChangableData;
   notifications: Notifications;
```
Comment (Ugmaxie) [inline src/my-profile/my-profile.component.ts:30]: should get value in init or constructor

File: src/shared/crop-image/crop-image.component.ts [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+})
+export class CropImageComponent {
+  @ViewChild('cropper', undefined) cropper: ImageCropperComponent;
+  @ViewChild('cropImgModal') cropImgModal: ModalDirective;
+  @Output() croppedFile: EventEmitter<UploadFile> = new EventEmitter();
+
+  isModalShown = false;
+  data: any;
+  cropperSettings: CropperSettings;
+  fileName: string;
```
Comment (Ugmaxie) [inline src/shared/crop-image/crop-image.component.ts]: should get value via init or constructor

**Review by Ugmaxie — CHANGES_REQUESTED** [review]

File: src/my-profile/my-profile.component.ts (hunk 1 above)
Comment (DiLiUA) [reply src/my-profile/my-profile.component.ts:16]: I'll do it for All project in a separate PR

File: src/my-profile/my-profile.component.ts (hunk 2 above)
Comment (DiLiUA) [reply src/my-profile/my-profile.component.ts:30]: notifications  initing in the Constructor

File: src/my-profile/my-profile.component.ts (hunk 1 above)
Comment (DiLiUA) [reply src/my-profile/my-profile.component.ts:16]: Created issue #74

**Review by Ugmaxie — APPROVED** [review]

**Pull request merged**